dockerfiles/reset_header_check_count/startup_script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# Starts the resetHeaderCheckCount command
+
+# Verify required args present
+MISSING_VAR_MESSAGE=" is required and no value was given"
+
+if test -z "$HEADER_BLOCK_NUMBER"
+then
+    echo HEADER_BLOCK_NUMBER $MISSING_VAR_MESSAGE
+    exit 1
+fi
+
+function testDatabaseVariables() {
```

Comment:
Thinking it may be nice to pull this out into some shared code, since how a db connection string is created shouldn't differ between commands, but haven't thought about where that code should live. Maybe just a helper in the dockerfiles directory. 🤔